tutorials/docker-run.mdx

```diff
@@ -0,0 +1,401 @@
+---
+title: 'Docker Run on SaladCloud'
+---
+
+This tutorial will guide you through the process of running a container locally and deploying a GPU-accelerated application on SaladCloud, following best practices. Before you begin, please complete [the quickstart guide](https://docs.salad.com/products/quickstart) to set up your account, organization and project on SaladCloud. This tutorial assumes you have a basic understanding of containers and images, Python, Docker commands, job queues, and load balancers.
```

Comment:
links to other docs pages should leave out the domain, so that they work locally and in staging environments.
```suggestion
This tutorial will guide you through the process of running a container locally and deploying a GPU-accelerated application on SaladCloud, following best practices. Before you begin, please complete [the quickstart guide](/products/quickstart) to set up your account, organization and project on SaladCloud. This tutorial assumes you have a basic understanding of containers and images, Python, Docker commands, job queues, and load balancers.
```

Reply:
Removed the domain for all the links to the doc site.